contracts/BatchAuction.sol

```diff
@@ -1,13 +1,23 @@
 pragma solidity ^0.5.0;
 
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
-contract BatchAuction {
+
+contract BatchAuction is Ownable {
 
     uint16 public constant MAX_ACCOUNT_ID = 100;
+    uint8 public constant MAX_TOKENS = 30;
 
     mapping (address => uint16) public publicKeyToAccountMap;
     mapping (uint16 => address) public accountToPublicKeyMap;
 
+    uint8 public numTokens;
+    mapping (address => uint8) public tokenAddresToIdMap;
```

Comment:
Maybe a single list with MAX_TOKENS slot is sufficient here?

Reply:
Although this wouldn't allow us to change Max Tokens in the future without redeploying the contract.

Reply:
And if we have a list we couldn't recover index from address without looping (I have already used the inverse for account mapping)